src/main/java/org/tailormap/api/persistence/helper/ApplicationHelper.java

```diff
@@ -229,6 +233,9 @@ private void addAppTreeNodeItem(AppTreeNode node, List<LayerTreeNode> layerTreeN
       LayerTreeNode layerTreeNode = new LayerTreeNode();
       if ("AppTreeLayerNode".equals(node.getObjectType())) {
         AppTreeLayerNode appTreeLayerNode = (AppTreeLayerNode) node;
+        boolean webMercatorAvailable = this.isWebMercatorAvailable(appTreeLayerNode);
```

Comment:
Waarom dit hier bepalen? Dat kan lijkt me ook in addAppLayerItem(), waar al findServiceLayer() is aangeroepen, en je dus al de GeoService en GeoServiceLayer hebt. Dus dan kan de isWebMercatorAvailable() methode ook simpeler.